Add hydrostatic column benchmark validation (#7)
cb-geo/mpm-doc: user/validation/explicit/hydrostatic-column/hydrostatic-column.md

> # Hydrostatic Column

Reviewer: Please use sentence case in titles

Author: Done

## Hydrostatic Analytical Solution

> The hydrostatic validation involves applying gravity loading to a column of material restrained in both lateral directions and along the bottom plane. The Newtonian fluid model will yield hydrostatic pressures for all normal stress directions ($\sigma_{xx}$, $\sigma_{yy}$, $\sigma_{zz}$). The Linear Elastic material will yield static gravity driven stress in vertical direction while the horizontal stress depends on the poisson's ratio.

Reviewer: ```suggestion
The hydrostatic validation involves applying gravity loading to a column of material restrained in both lateral directions and along the bottom plane. The Newtonian fluid model will yield hydrostatic pressures for all normal stress components ($\sigma_{xx}$, $\sigma_{yy}$, $\sigma_{zz}$). The linear elastic material will yield geo-static stresses in the vertical direction and the horizontal stresses depend on the Poisson's ratio.
```

![hydrostatic column](hydrostatic-column.png)

> *Hydrostatic column with a width `w` of 0.2 m and height `h` of 0.1 m. Restrained in both the lateral directions and along the bottom plane*

Reviewer: Use `>` for fig caption

## MPM configuration

### Mesh

> 

Reviewer: Could you align these columns with spaces?

Author: Done!

|Particles per cell |  25  |  25  |  25  |

### Analysis

|Description		| Case 1		| Case 2		|  Case 3		|
|-----------------------|---------------|---------------|---------------|
|Total analysis time 	| 0.2 s		| 0.2 s		| 0.2 s		|
|dt                     | 0.00001 s | 0.000005 s |0.0000025 s |
|Gravity		| -9.81 $m/s^2$		| -9.81 $m/s^2$		| -9.81 $m/s^2$		|

### Material

|Description		| value		|
|-----------------------|---------------|
|Material	                          | Newtonian |
|Density ($\rho$) 		              | 1800.0 $kg/m^3$ |
|Bulk modulus ($K$)	                  | 1000000 $N/m^2$	|
|Viscosity ($\mu$)                    |  0.0     |

|Description		| value		|
|-----------------------|---------------|
|Material	                          | Linear Elastic |
|Density ($\rho$) 		              | 1800.0 $kg/m^3$ |
|Young's modulus ($E$)	              | 1000000 $N/m^2$	|
|Poisson's ratio ($\nu$)              |  0.0     |

## Hydrostatic Analysis

Analysis are carried out using MPM Explicit USF and USL algorithms using velocity update. Note that the results converge with more refined mesh with smaller errors.

## Results for Newtonian Fluid

> ### USF Results at 0.2s

Reviewer: ```suggestion
### USF results at t = 0.2s
```

| Parameter				| Analytical	| Case 1	| Case 2 	| Case 3 	|
|-----------------------|---------------|---------------|---------------|---------------|
|$\sigma_{yy} (N/m^2)$			| -1765.800		    | -1680.244	| -1724.583 | -1746.791 |
|$\sigma_{xx} (N/m^2)$          | -1765.800 		| -1680.244	| -1724.583 | -1746.791 |

### USL Results at 0.2s

| Parameter				| Analytical	| Case 1	| Case 2 	| Case 3 	|
|-----------------------|---------------|---------------|---------------|---------------|
|$\sigma_{yy} (N/m^2)$			| -1765.800		    | -1680.182	| -1724.550 | -1746.774 |
|$\sigma_{xx} (N/m^2)$          | -1765.800 		| -1680.182	| -1724.550 | -1746.774 |

> ## Results for Linear Elastic

Reviewer: ```suggestion
## Results (Linear Elastic)
```

### USF results at t = 0.2s

| Parameter				| Analytical 	| Case 1	| Case 2 	| Case 3 	|
|-----------------------|---------------|-----------|-----------|-----------|
|$\sigma_{yy} (N/m^2)$	| -1765.800	    | -1680.244	| -1724.583 | -1746.791 |
|$\sigma_{xx} (N/m^2)$  |     0.000	    |     0.000	|     0.000 |  0.000    |

### USL results at t = 0.2s

| Parameter				| Analytical 	| Case 1	| Case 2 	| Case 3 	|
|-----------------------|---------------|-----------|-----------|-----------|
|$\sigma_{yy} (N/m^2)$	| -1765.800		| -1680.182	| -1724.550 | -1746.774 |
|$\sigma_{xx} (N/m^2)$  |     0.000  	|     0.000	|     0.000 |  0.000    |

> ## Plot over time to show convergence

Reviewer: Remove this title

> ## Plot over time to show convergence

Reviewer: ```suggestion
```

![convergence](convergence.png)

> > There are some oscillations that finally converge

Reviewer: ```suggestion
> Plot of vertical stresses with time showing convergence of stress osciallations
```

> ## Results image 

Reviewer: ```suggestion
```

> ![result-image](result-image.png)

Reviewer: Could you rename this file with a meaningful name?

Reviewer: linear-elastic-stresses.png

> ![result-image](result-image.png)

Reviewer: ```suggestion
![linear-elastic-stresses](linear-elastic-stresses.png)
```

> ![animation](animation.gif)

Reviewer: Could we just have a snapshot rather than an animation, it won't work very well on a PDF. If you like to have a video, let's create a proper video and upload to YouTube / Vimeo

Author: I will make a snapshot of the last one then

> # Hydrostatic Column

Reviewer: ```suggestion
# Hydrostatic column
```

> 

Reviewer: Remove empty lines

> ## Hydrostatic Analytical Solution

Reviewer: ## Hydrostatic analytical solution
```suggestion
## Analytical solution
```

|Particles per cell |  25  |  25  |  25  |

### Analysis

|Description		| Case 1		| Case 2		|  Case 3		|
|-----------------------|---------------|---------------|---------------|
|Total analysis time 	| 0.1 s		| 0.1 s		| 0.1 s		|
|dt                     | 0.00001 s | 0.000005 s |0.0000025 s |
|Gravity		| -9.81 $m/s^2$		| -9.81 $m/s^2$		| -9.81 $m/s^2$		|

### Material

|Description		| value		|
|-----------------------|---------------|
|Material	                          | Newtonian |
|Density ($\rho$) 		              | 1800.0 $kg/m^3$ |
|Bulk modulus ($K$)	                  | 1000000 $N/m^2$	|
|Viscosity ($\mu$)                    |  0.0     |

|Description		| value		|
|-----------------------|---------------|
|Material	                          | Linear Elastic |
|Density ($\rho$) 		              | 1800.0 $kg/m^3$ |
|Young's modulus ($E$)	              | 1000000 $N/m^2$	|
|Poisson's ratio ($\nu$)              |  0.0     |

## Hydrostatic Analysis

Analysis are carried out using MPM Explicit USF and USL algorithms using velocity update. Note that the results converge with more refined mesh with smaller errors.

## Results for Newtonian Fluid

> 

Reviewer: Could you show a plot of stress convergence with time?

Author: convergence is shown now

### USF Results at 0.1s

| Parameter				| Analytical	| Case 1	| Case 2 	| Case 3 	|
|-----------------------|---------------|---------------|---------------|---------------|
|$\sigma_{yy} (N/m^2)$			| -1765.800		    | -1680.244	| -1724.627 | -1741.565 |
|$\sigma_{xx} (N/m^2)$          | -1765.800 		| -1680.244	| -1724.627 | -1741.565 |

### USL Results at 0.1s

| Parameter				| Analytical	| Case 1	| Case 2 	| Case 3 	|
|-----------------------|---------------|---------------|---------------|---------------|
|$\sigma_{yy} (N/m^2)$			| -1765.800		    | -1680.182	| -1724.594 | -1741.351 |
|$\sigma_{xx} (N/m^2)$          | -1765.800 		| -1680.182	| -1724.594 | -1741.351 |

> ## Results for Linear Elastic

Reviewer: Could you show a plot of stress convergence with time?

### USF Results at 0.1s

| Parameter				| Analytical 	| Case 1	| Case 2 	| Case 3 	|
|-----------------------|---------------|---------------|---------------|---------------|
|$\sigma_{yy} (N/m^2)$			| -1765.800		    | -1680.244	| -1724.627 | -1741.565 |
|$\sigma_{xx} (N/m^2)$          |     0.000		    |     0.000	|    -2.327E-9 |  0.000 |

### USL Results at 0.1s

| Parameter				| Analytical 	| Case 1	| Case 2 	| Case 3 	|
|-----------------------|---------------|---------------|---------------|---------------|
|$\sigma_{yy} (N/m^2)$			| -1765.800		    | -1680.182	| -1724.594 | -1741.351 |
|$\sigma_{xx} (N/m^2)$          |     0.000  		|     0.000	|    -1.191E-9 |  0.000 |

## Results Animation

> ![results](results.gif)

Reviewer: results.gif is too large

Author: I tried to reduce to 3.7 MB. Do you want it to be less than 2 MB?
